remove direct placement (#7400)
pingcap/docs: sql-statements/sql-statement-alter-placement-policy.md

`ALTER PLACEMENT POLICY` is used to modify existing placement policies that have previously been created. All the tables and partitions which use the placement policy will automatically be updated.

> `ALTER PLACEMENT POLICY` _replaces_ the previous policy with the new definition. It does not _merge_ the old policy with the new one. In the following example, `FOLLOWERS=4` is lost when the `ALTER PLACEMENT POLICY` is executed:

Reviewer: ```suggestion
`ALTER PLACEMENT POLICY` _replaces_ the previous policy with the new definition. It does not _merge_ the old policy with the new one. In the following example, `FOLLOWERS=4` is overwritten by the `ALTER PLACEMENT POLICY` statement:
```

Author: "lost" is more clear here, because it was not specified (overwritten) in the ALTER statement.

> `ALTER PLACEMENT POLICY` _replaces_ the previous policy with the new definition. It does not _merge_ the old policy with the new one. i.e. in the following example `FOLLOWERS=4` is lost when the `ALTER PLACEMENT POLICY` is executed:

Reviewer: ```suggestion
`ALTER PLACEMENT POLICY` _replaces_ the previous policy with the new definition. It does not _merge_ the old policy with the new one. In the following example, `FOLLOWERS=4` is lost when the `ALTER PLACEMENT POLICY` is executed:
```
